### PR TITLE
Test/chalkboard redux tests

### PR DESCRIPTION
**Pull request by lilianaweimer**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
 
 
Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [x] Testing
- [ ] Other (please describe): 

## What is the current behavior?

- `<Chalkboard>` tests were failing for unknown and strange reasons

## What is the new behavior?

- `<Chalkboard>` tests are all passing
- Added a test to `<App>`

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: src/App/App.test.js [hunk 1]
```diff
@@ -1,12 +1,48 @@
...
-import { createStore } from 'redux'
+import { createStore, applyMiddleware } from 'redux'
 import { BrowserRouter } from 'react-router-dom'
-import { Provider } from 'react-redux'
+import { Provider } from 'react-redux';
+import { rootReducer } from '../reducers/index';
+import thunk from 'redux-thunk';
 
-const store = createStore(() => ({ isLoading: false, hasErrored: ''}))
+const store = createStore(rootReducer, {
```
Comment (Billwilke42) [inline src/App/App.test.js:11]: Amazing job on this, it was so hard!

File: src/Chalkboard/testHelper.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
```
Comment (Billwilke42) [inline src/Chalkboard/testHelper.js:1]: Great job on this test suite

File: src/Chalkboard/Chalkboard.test.js [hunk 3]
```diff
@@ -86,47 +73,72 @@ describe('Chalkboard', () => {
     });
 
     it('should increase score when an answer is correct', () => {
-      const { getByText, debug } = render(<Chalkboard />);
+      const { getByText } = render(<MemoryRouter><Provider store={store}><Chalkboard /></Provider></MemoryRouter>);
```
Comment (Billwilke42) [inline src/Chalkboard/Chalkboard.test.js:76]: Maybe we should try setting our components to a variable in our test suites to save space?

File: src/App/App.test.js [hunk 4]
```diff
@@ -1,12 +1,48 @@
...
+      "General Motors",
+      "Fiat"
+    ]
+    }
+]],
+  setCurrentRound: 0,
+  setCurrentQuestion: 0,
+  setScore: 0,
+  setLives: 3,
+}, applyMiddleware(thunk))
```
Comment (KyleWong2510) [inline src/App/App.test.js:45]: Impressive...I would have been so lost.  Great job finding the answers!

File: src/Chalkboard/Chalkboard.test.js [hunk 5]
```diff
@@ -1,68 +1,55 @@
...
+            "Fiat"
+          ]
+          }
+      ]],
+        setCurrentRound: 0,
+        setCurrentQuestion: 0,
+        setScore: 0,
+        setLives: 3,
+      })
+    })
```
Comment (KyleWong2510) [inline src/Chalkboard/Chalkboard.test.js:49]: Nice use of the beforeEach

File: src/Chalkboard/Chalkboard.test.js [hunk 6]
```diff
@@ -86,47 +73,72 @@ describe('Chalkboard', () => {
...
       fireEvent.click(answer);
-      
+
+      const newLives = getAllByTestId('heart');
+      expect(newLives.length).toBe(2);
     });
 
     it('should not decrease lives when an answer is correct', () => {
-      const { getByText, debug } = render(<Chalkboard />);
+      const { getByText, getAllByTestId } = render(<MemoryRouter><Provider store={store}><Chalkboard /></Provider></MemoryRouter>);
```
Comment (KyleWong2510) [inline src/Chalkboard/Chalkboard.test.js:116]: could renders also be declared in the beforeEach?

File: src/Chalkboard/testHelper.js [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+    decrementScore: decrementScore
+    })),
+    ...renderOptions
+  } = {}
+) => {
+  function Wrapper({ children }) {
+    return <MemoryRouter><Provider store={store}>{children}</Provider></MemoryRouter>
+  }
+  return rtlRender(ui, { wrapper: Wrapper, ...renderOptions })
+}
```
Comment (KyleWong2510) [inline src/Chalkboard/testHelper.js:64]: so crazy that we have to do this

**Pull request merged**